.circleci/config.yml

```diff
@@ -148,6 +148,9 @@ workflows:
       - prep-build-confirmation-redesign-test-mv2:
```

Comment:
We're going to avoid using new flagged jobs; instead, we'll now use Developer Options:

https://consensys.slack.com/archives/CTQAGKY5V/p1717683404677029?thread_ts=1717679382.264779&cid=CTQAGKY5V